testing.py

```diff
@@ -438,6 +438,10 @@ def test_name(self):
             self.print_database_instance()  # TODO: remove before prod
             [code...]
         """
+        if not 'TESTING' in os.environ:
```

Comment:
I suggest you use logging.error() or logging.warn() here to print out a blatant warning so that we notice the code is in place and know to remove it.